e2e-tests/cypress/tests/support/index.js

```diff
@@ -216,6 +216,9 @@ function sysadminSetup(user) {
     // # Deactivate test bots if any
     cy.apiDeactivateTestBots();
 
+    // # Disable welcome tours if any
+    cy.apiDisableTutorials(user.id);
```

Comment:
@saturninoabril I added this to make the smoke tests succeed, as the tooltips for insights were showing up for the admin user. But please let me know if there are better ways to solve this

Reply:
Looks good to me, thanks!